releasenotes/notes/update-qpdgate-labels-612e7d355ae1c1a3.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrades:
```

Comment:
Kinda weird to put a breaking change in "upgrades", but it is a code quality upgrade, and it is in line with [reno docs](https://docs.openstack.org/reno/latest/user/usage.html).

```
deprecations
A list of features, APIs, configuration options to be deprecated in the release. Deprecations should not be used for something that is removed in the release, use upgrade section instead. Deprecation should allow time for users to make necessary changes for the removal to happen in a future release.
```